src/common/queries/expenses.ts

```diff
@@ -70,14 +71,15 @@ export function useExpensesQuery(params: ExpensesParams) {
       request(
         'GET',
         endpoint(
-          '/api/v1/expenses?filter=:filter&per_page=:per_page&status=:status&page=:page&match_transactions=:match_transactions&includes=:includes',
+          '/api/v1/expenses?filter=:filter&per_page=:per_page&status=:status&page=:page&match_transactions=:match_transactions&includes=:includes&include=:include',
```

Comment:
why do we have `includes=:includes`

i don't think is a valid parameter at all?

Reply:
@turbo124 Yeah, I noticed the same thing. I didn't want to touch anything there because I didn't know if it's a valid parameter or not. Let me know if I should remove it.

Reply:
@turbo124 We never use this param so it is removed.